src/common/utils/log/logger.go

```diff
@@ -93,6 +93,14 @@ func (l *Logger) SetLevel(lvl Level) {
 	l.lvl = lvl
 }
 
+//SetSkipLine sets the skipLine of Logger l
+func (l *Logger) SetSkipLine(skipLine bool) {
```

Comment:
The `skipLine` is an internal variable, we shouldn't expose it. 
Even though we call the method to set `skipLine` as `true`, the `Debug()`, `Info()` and `Warning()` don't print the information about line either.

Reply:
For 1, the `skipLine` is in the same scope with the test codes, you can just set it to `true` by calling `logger.skipLine = true` directly. 
And I think maybe we shouldn't set it to `true`, we should test it for the real usage.
For 2, I don't think it's the issue that Harbor should fix. The log collector shouldn't require that all the applications are using the same log format.

Reply:
Fixed, thanks.